platform/darwin/src/MGLOfflineStorage.mm

```diff
@@ -341,7 +340,7 @@ - (void)_addContentsOfFile:(NSString *)filePath withCompletionHandler:(void (^)(
         NSMutableArray *packs;
         if (!result) {
             NSString *description = [NSString stringWithFormat:NSLocalizedStringWithDefaultValue(@"ADD_FILE_CONTENTS_FAILED_DESC", @"Foundation", nil, @"Unable to add offline packs from the file at %@.", @"User-friendly error description"), filePath];
-            error = [NSError errorWithDomain:MGLErrorDomain code:-1 userInfo:@{
+            error = [NSError errorWithDomain:MGLErrorDomain code:9 userInfo:@{
```

Comment:
Thanks for adding specific error codes for these. Instead of using the numeric code directly, use the `MGLErrorCodeModifyingOfflineStorageFailed` constant you added in `MGLTypes.h`.